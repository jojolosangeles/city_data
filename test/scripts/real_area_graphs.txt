load ../data/real/Traffic_Collision_Data_from_2010_to_Present.csv as all_areas
columns all_areas
drop all_areas DR+Number
drop all_areas Date+Reported
drop all_areas Area+ID
drop all_areas Reporting+District
drop all_areas Crime+Code
drop all_areas Crime+Code+Description
drop all_areas MO+Codes
drop all_areas Premise+Code
drop all_areas Premise+Description
drop all_areas Location
create all_areas Date [pd.to_datetime(x) for x in dataFrame['Date Occurred']]
create all_areas Weekday [x.weekday() for x in dataFrame['Date']]
create all_areas Year [x.year for x in dataFrame['Date']]
save all_areas
