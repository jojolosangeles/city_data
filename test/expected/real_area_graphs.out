Test: scripts/real_area_graphs.txt
> load ../data/real/Traffic_Collision_Data_from_2010_to_Present.csv as all_areas
> columns all_areas
['DR Number' 'Date Reported' 'Date Occurred' 'Time Occurred' 'Area ID'
 'Area Name' 'Reporting District' 'Crime Code' 'Crime Code Description'
 'MO Codes' 'Victim Age' 'Victim Sex' 'Victim Descent' 'Premise Code'
 'Premise Description' 'Address' 'Cross Street' 'Location']
> drop all_areas DR Number, Date Reported, Area ID, Reporting District, Crime Code
Dropping ['DR Number', 'Date Reported', 'Area ID', 'Reporting District', 'Crime Code'] from 'all_areas'
> drop all_areas Crime Code Description, MO Codes, Premise Code, Premise Description, Location
Dropping ['Crime Code Description', 'MO Codes', 'Premise Code', 'Premise Description', 'Location'] from 'all_areas'
> create all_areas Date [pd.to_datetime(x) for x in dataFrame['Date Occurred']]
all_areas.Date=
> create all_areas Weekday [x.weekday() for x in dataFrame['Date']]
all_areas.Weekday=
> create all_areas Year [x.year for x in dataFrame['Date']]
all_areas.Year=
> save all_areas
Saved all_areas.csv
Test completed: scripts/real_area_graphs.txt
